README.md

```diff
@@ -134,6 +134,7 @@ ML Ops|[Zilliz](https://zilliz.com)'s [Towhee](https://towhee.io)|Platform for g
 Messaging|[Element](https://element.io/)|Enterprise communication platform|<a href=https://github.com/vector-im/element-web><img src="https://img.shields.io/github/stars/vector-im/element-web?style=social" width=150/></a>|[Slack](https://slack.com/)|
 Messaging|[Mattermost](https://mattermost.com/)|Enterprise communication platform for developers|<a href=https://github.com/mattermost/mattermost-server><img src="https://img.shields.io/github/stars/mattermost/mattermost-server?style=social" width=150/></a>|[Slack](https://slack.com/)|
 Messaging|[Rocket.chat](https://rocket.chat/)|Enterprise communication platform|<a href=https://github.com/RocketChat/Rocket.Chat><img src="https://img.shields.io/github/stars/RocketChat/Rocket.Chat?style=social" width=150/></a>|[Slack](https://slack.com/)|
+Messaging|[Tinode](https://tinode.co/)|General instant messaging|<a href=https://github.com/tiode/chat><img src="https://img.shields.io/github/stars/tinode/chat?style=social" width=150/></a>|[WhatsApp](https://www.whatsapp.com/), [Telegram](https://www.telegram.org/)|
```

Comment:
https://github.com/tiode/chat 404，should https://github.com/tinode/chat ？

Reply:
Yes, it should be https://github.com/tinode/chat